### ISSUE TITLE
JS errors when trying to edit an image. 

### ISSUE THREAD

**Issue by pooja-muchandikar** [rtCamp/rtMedia#1689]
Refer the visual-> https://drive.google.com/file/d/1dY8kGXw8dlojNy34_Vp884X-i69elO2W/view

While trying to edit the image and modify the image JS error is shown. 

```
Test Environment
rtMedia: v4.6.4
BuddyPress: v6.2.0
```

**Comment by abhijitrakas** [comment]
@pooja-muchandikar - Added fix in `issue/gh-1689-console-errors` branch. Please test this branch then will create PR.

**Comment by pooja-muchandikar** [comment]
@abhijitrakas 

Cross-checked the issue with `issue/gh-1689-console-errors` branch. And found that modify image button is working and redirecting to modify image page but still in console JS errors are shown. Please check the attached visual. 

Reference Visual: https://drive.google.com/file/d/1dv1hQ6GPQQiDMxgA-gizAc3p-qIT6nNV/view

```
Test Environment
rtMedia: v4.6.4 issue/gh-1689-console-errors branch
BuddyPress: v6.3.0 Both legacy and nouveau template
Theme: 2017, 2019, 2020
```

**Note: Checked with both profile media and group media**

**Comment by abhijitrakas** [comment]
@pooja-muchandikar - Updated code and fixed issue. Please test the same branch again.

**Comment by pooja-muchandikar** [comment]
This issue is not yet fixed. Checked with the latest changes but still error is shown in the console. 

@ravatparmar please check this issue.

**Comment by ravatparmar** [comment]
@pooja-muchandikar 

Can you please check the above issue, and can you please let me know if you are able to reproduce it.

Thanks.

**Comment by ravatparmar** [comment]
The above issue was related to Query Monitor.

CC: @pooja-muchandikar

**Comment by pooja-muchandikar** [comment]
Alright Noted. 

The main issue mentioned [here](https://github.com/rtCamp/rtMedia/issues/1689#issue-701636418) is fixed. 

@ravatparmar  The fix is available in another branch,Can you please merged the fix in `pre-release`?

**Comment by ravatparmar** [comment]
Yes

**Comment by pooja-muchandikar** [comment]
Tested with `pre-release` branch and fix added works as expected.

**Comment by pooja-muchandikar** [comment]
Tested with `master` branch and the issue is fixed.

**Comment by pooja-muchandikar** [comment]
Released with `v4.6.5`
